game/scripts/npc/heroes/broodmother.txt

```diff
@@ -7,6 +7,11 @@
   {
     // Abilities
     //-------------------------------------------------------------------------------------------------------------
-    "Ability13"   "special_bonus_attack_damage_45" // replaces special_bonus_agility_20
+	"AttributeAgilityGain"		"3.40000"
+	"AttributeStrengthGain"		"3.00000" 
+	"AttributeBaseAgility"     "25"
```

Comment:
Set base agility to 18 please.

Reply:
Broodmother got base damage buff several patches in a row. We don't want another hero to hit like a truck in early game.